### PR TITLE
update parameters.py

### PR DESCRIPTION
**Pull request by WuShichao**
@ahnitz According to your good suggestion, I separated parts of the code from https://github.com/gwastro/pycbc/pull/4289 as separated PRs, this PR updates several parameter names for LISA.

### CODE REVIEW

File: pycbc/waveform/parameters.py [hunk 1]
```diff
@@ -417,32 +417,34 @@ def docstr(self, prefix='', include_label=True):
...
 delta_tc = Parameter("delta_tc", dtype=float,
                      label=r"$\Delta t_c~(\rm{s})$",
-                     description="Coalesence time offset.")
+                     description="Coalesence time offset in " +
+                     "geocentric/SSB/LISA coords.")
 ra = Parameter("ra",
-                dtype=float, default=0., label=r"$\alpha$",
-                description="Right ascension (rad).")
+               dtype=float, default=0., label=r"$\alpha$",
+               description="Right ascension (rad) in geocentric coords.")
```
Comment (ahnitz) [inline pycbc/waveform/parameters.py:429]: It doesn't make sense to say that ra/dec are in geocentric coordinates. They are uniquely defined, so there is no extra coordinate freedom.

File: pycbc/waveform/parameters.py [hunk 2]
```diff
@@ -417,32 +417,34 @@ def docstr(self, prefix='', include_label=True):
                 dtype=float, default=0., label=r"$\delta$",
                 description="Mean anomaly of the periastron (rad).")
 tc = Parameter("tc",
-                dtype=float, default=None, label=r"$t_c$ (s)",
-                description="Coalescence time (s).")
+               dtype=float, default=None, label=r"$t_c$ (s)",
+               description="Coalescence time (s) in "
+                           "geocentric/SSB/LISA coords.")
```
Comment (ahnitz) [inline pycbc/waveform/parameters.py:422]: @WuShichao Looking at this, I'd take out reference here to coordinates systems generally. I think it's going to end up being confusing.

**Review by ahnitz — CHANGES_REQUESTED** [review]
See minor request, once that is taken care of this can be merged.

**Comment by WuShichao** [comment]
@ahnitz Poke :)

File: pycbc/waveform/parameters.py [hunk 3]
```diff
@@ -417,32 +417,35 @@ def docstr(self, prefix='', include_label=True):
...
-                dtype=float, default=None, label=r"$t_c$ (s)",
-                description="Coalescence time (s).")
+               dtype=float, default=None, label=r"$t_c$ (s)",
+               description="Coalescence time (s) is the time when a GW "
+                           "reaches the origin of a certain coordinate system.")
 delta_tc = Parameter("delta_tc", dtype=float,
                      label=r"$\Delta t_c~(\rm{s})$",
-                     description="Coalesence time offset.")
+                     description="Coalesence time offset in " +
+                     "a certain coordinate system.")
```
Comment (ahnitz) [inline pycbc/waveform/parameters.py:426]: Drop 'in a certain coordinate system'. I don't think that adds any clarification.

Comment (WuShichao) [reply pycbc/waveform/parameters.py]: @ahnitz Yeah, that makes sense. I have removed it.

**Review by ahnitz — APPROVED** [review]

**Pull request merged**